leo/updater.rs

```diff
@@ -27,7 +28,7 @@ impl Updater {
     const LEO_REPO_OWNER: &'static str = "AleoHQ";
 
     /// Show all available releases for `leo`.
-    pub fn show_available_releases() -> Result<(), UpdaterError> {
+    pub fn show_available_releases() -> Result<(), Error> {
```

Comment:
Does this change to `anyhow::Error` produce the same `UpdaterError` message in the terminal as before (if any)?

Reply:
Here are the outputs (when no network available):

Current version:
```
Updating ReqwestError: error sending request for url (https://api.github.com/repos/AleoHQ/leo/releases/latest): error trying to connect: dns error: failed to lookup address information: nodename nor servname provided, or not known

  Updating Could not update Leo to the latest version
  Updating self_update: ReqwestError: error sending request for url (https://api.github.com/repos/AleoHQ/leo/releases/latest): error trying to connect: dns error: failed to lookup address information: nodename nor servname provided, or not known
```

This update:
```
Updating Could not update Leo to the latest version
  Updating ReqwestError: error sending request for url (https://api.github.com/repos/AleoHQ/leo/releases/latest): error trying to connect: dns error: failed to lookup address information: nodename nor servname provided, or not known
```